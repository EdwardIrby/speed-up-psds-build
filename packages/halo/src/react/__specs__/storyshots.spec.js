import path from 'path'
import React from 'react'
import initStoryshots, {
  snapshotWithOptions
} from '@storybook/addon-storyshots'

jest.mock('../__stories__/focusable.js', () => props => <div {...props} />)

const createNodeMock = () => document.createElement('div')

initStoryshots({
  configPath: path.resolve(__dirname, '../../../.storybook'),
  framework: 'react',
  test: snapshotWithOptions({ createNodeMock })
})
