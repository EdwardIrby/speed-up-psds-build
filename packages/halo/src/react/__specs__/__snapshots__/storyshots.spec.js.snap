// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Halo/error false 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-1xm24a9=""
  >
    <div>
      not error
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/error true 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-utxo7p=""
  >
    <div>
      error
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/gapSizes default 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-nbiptv=""
  >
    <div>
      default
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/gapSizes small 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-nbiptv=""
  >
    <div>
      small
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/shapes default 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-nbiptv=""
  >
    <div
      shape="default"
    >
      default
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/shapes pill 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-34tmci=""
  >
    <div
      shape="pill"
    >
      pill
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/visible false & error 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-utxo7p=""
  >
    <div>
      not visible (always visible if error) & error
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/visible false & not visibleOnFocus 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-14u1z9b=""
  >
    <div>
      really not visible (even on focus)
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/visible false 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-1xm24a9=""
  >
    <div>
      not visible (until focused)
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/visible true & error 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-nbiptv=""
  >
    <div>
      visible & error
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/visible true 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-nbiptv=""
  >
    <div>
      visible
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/visibleOnFocus enabled & error 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-utxo7p=""
  >
    <div>
      visible when focused with error
    </div>
  </div>
</div>
`;

exports[`Storyshots Halo/visibleOnFocus enabled 1`] = `
<div
  data-css-claeju=""
>
  <div
    data-css-1xm24a9=""
  >
    <div>
      visible when focused
    </div>
  </div>
</div>
`;
